tests/src/core/ArrayVectorTest.cpp

```diff
@@ -826,6 +825,7 @@ private slots:
 		QVERIFY(!(e != v));
 		QVERIFY(g != v);
```

Comment:
Just as a verification, can you change the `!=` to `==` and see if `ctest` marks this one as failed (unless you already did something like this)?

Reply:
I verified it. `ArrayVectorTest` fails as expected, but because of some odd rpmalloc leak, `AutomatableModelTest` and `AutomationTrackTest` fail due to an assertion at the very end. When I ran this PR, and as you can see in the CI, nothing was wrong. I really dont know why this happens, but the tests themselves actually pass.

I had plans to investigate this and fix it, but when I did, it disappeared! I can't accurately know if I'm messing up somewhere or not, but the best I can do is temporaily hook the tests up with AddressSanitizer and use `malloc` instead of `rpmalloc`.